tasklist/client/src/common/tasks/details/TaskDetailsHeader/index.tsx

```diff
@@ -10,15 +10,16 @@ import {useTranslation} from 'react-i18next';
 import {Stack} from '@carbon/react';
 import {CheckmarkFilled} from '@carbon/react/icons';
 import {AssigneeTag} from 'common/components/AssigneeTag';
-import type {CurrentUser} from '@vzeta/camunda-api-zod-schemas/8.8';
+import type {CurrentUser, UserTask} from '@vzeta/camunda-api-zod-schemas/8.8';
 import styles from './styles.module.scss';
 import taskDetailsLayoutCommon from 'common/tasks/details/taskDetailsLayoutCommon.module.scss';
+import {TaskStateLoadingText} from 'common/tasks/details/TaskStateLoadingText';
 
 type Props = {
   taskName: string;
   processName: string;
   assignee: string | null;
-  taskState: 'CREATED' | 'COMPLETED' | 'CANCELED' | 'FAILED';
+  taskState: UserTask['state'];
```

Comment:
I wrote the states manually here because this component is used on V1 and V2 and I would like to avoid referencing types specific to aversion

Let's just manually type for now. On 8.10 we can migrate to use the V2 types